packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -96,6 +96,19 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         : XAxisTypes.StringAxis;
   }
 
+  public componentDidMount(): void {
+    const isChartEmpty = d3Min(this._points, (point: IVerticalBarChartDataPoint) => point.y) === 0;
```

Comment:
what if there are 2 points in the chart. One with y=0 and other with y=10. d3 min will still return 0, right?

Reply:
yes changed it to d3Max <= 0